src/editor.c

```diff
@@ -4460,12 +4460,12 @@ void editor_set_font(GeanyEditor *editor, const gchar *font)
 	g_return_if_fail(editor);
 
 	pfd = pango_font_description_from_string(font);
-	size = pango_font_description_get_size(pfd) / PANGO_SCALE;
+	size = (100*pango_font_description_get_size(pfd)) / PANGO_SCALE;
```

Comment:
why multiply by 100 and divide later?  wouldn't it be simpler and everything to just make `size` a `gdouble` and get a real straight away?